README.md

```diff
@@ -1,137 +1,221 @@
+  ```bash
+  npm run docker:local-db
+  ```
 
-![Superformula-front-end-test-mockup-2](./mockup2.png)
+  | *NOTE* Before running this command, ensure you have installed and running docker in your machine.
 
-> [Source Figma file](https://www.figma.com/file/hd7EgdTxJs2fpTzzSKlNxo/Superformula-full-stack-test)
+- **Unit Test**
 
-- Client will be performing real-time search against this API
-- List of users should be updated automatically after single user is updated
+  ```bash
+  npm run test:unit
+  ```
+- **Integration Test**
 
-## What We Care About
+  ```bash
+  npm run test:integration
+  ```
 
-Use any libraries that you would normally use if this were a real production App. Please note: we're interested in your code & the way you solve the problem, not how well you can use a particular library or feature.
+- **e2e Test**
+  ```bash
+  npm run test:e2e
+  ```
+  | *NOTE* There is an existing issue related to the background services after running the e2e are not killed properly. So, the work aroud, until the bug is fixed, will requires to kill it manually or restarting the machine. 
 
-_We're interested in your method and how you approach the problem just as much as we're interested in the end result._
 
-Here's what you should strive for:
+### Deployments
 
-- Good use of current Typescript, Node.js, GraphQL & performance best practices.
-- Solid testing approach.
-- Extensible code and architecture.
-- Delightful experience for other backend engineers working in this repository
-- Delightful experience for engineers consuming your APIs
+- **local**
 
-## Q&A
-> Where should I send back the result when I'm done?
+  ```bash
+  npm run local
+  ```
 
-Fork this repo and send us a pull request when you think you are done. There is no deadline for this task unless otherwise noted to you directly.
+- **Explorer and play with the service**
+
+  The GraphQL Playgraound is enabled in non prod environment to lets you explore the schema and execute queries a mutations
+  After executing "npm run local" you will be able to access to the Playgroind tool on http://localhost:3000/dev/playground
+
+  | *NOTE* Add the following header attribute on the "HATTP HEADERS" secition of the Playgraoung tool . 
+  ```js
+      {
+        "x-api-key": "super-formula-api-key-token"
+      }
+  ```
+
+- **aws**
+
+  - **aws cli setup**
+
+    ```bash
+    aws configure
+    ```
+
+    | *NOTE* Please configure your aws cli in this way only avoing to include the key in this proyect code.
+
+  - **deploy**
+
+    The following command will create and s3 in aws. package your and upload your service to it, create the api gateway and the Dynamdb database. This also connects these resources to aws cloudwatch monitoring service
+
+    ```bash
+    npm run deploy
+    ```
+
+    At the end of the process you will see the url for the api gateway that epxose this graphql lambda service. Remeber to add the header key
+
+  - **destroy**
+    ```bash
+    npm run deploy
```

Comment:
Should we execute `npm run deploy` to destroy the infrastructure?

Reply:
It should be 'npm run destroy'
Fixed